cloudfunctions/candidate_gmail_messages/cloudfunction.go

```diff
@@ -308,6 +323,57 @@ func (cf *CloudFunction) proccessMessages(messageIDs []string) error {
 			continue
 		}
 
+		parseRequest := ml.ParseJobRequest{
+			From:    srcmessage.Sender(message),
+			Subject: srcmessage.Subject(message),
+			Body:    srcmessage.Body(message),
+		}
+
+		log.Printf("parsing email: %s", message.Id)
+		job, err := parser.ParseJob(&parseRequest)
+		// for now, abort on error
+		if err != nil {
+			return err
+		}
+
+		// if fields are missing, skip
+		if job.Company == "" || job.Title == "" || job.Recruiter == "" {
+			// print sender and subject
+			log.Printf("skipping job: %v", job)
+			continue
+		}
+		recruiterEmail := srcmessage.SenderEmail(message)
+		data := map[string]interface{}{
+			"recruiter":       job.Recruiter,
+			"recruiter_email": recruiterEmail,
+		}
+
+		// turn data into json.RawMessage
+		b, err := json.Marshal(data)
+		if err != nil {
+			return err
+		}
+
+		// convert epoch ms to time.Time
+		emailedAt := time.Unix(message.InternalDate/1000, 0)
+
+		err = cf.queries.InsertUserEmailJob(cf.ctx, db.InsertUserEmailJobParams{
+			UserID: cf.user.UserID,
+			// is this correct? `profile.EmailAddress` in populate_jobs
+			UserEmail:     cf.user.Email,
```

Comment:
@devstein not sure if this is the correct email

Reply:
Good question. In the middle of a refactor to allow users to connect multiple emails, so we want to use the email associated with the messages rather than the account (`cf.user`). We can get this from 

```go
profile, err := cf.srv.Profile()
profile.EmailAddress
```